### PR TITLE
tests: implement simple end2end test

### PR DESCRIPTION
**Pull request by iejMac**

### CODE REVIEW

**Comment by iejMac** [comment]
@rom1504 Any suggestions on what to add?

File: tests/test_main.py [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+import pandas as pd
+import pytest
+import tarfile
+
+from video2dataset.main import video2dataset
+
+
+def test_e2e():
+    url_list = "tests/test_files/test_webvid.csv"
+    destination = "tests/dataset"
```
Comment (rom1504) [inline tests/test_main.py:10]: I advise using a temporary dir using tempdir lib
makes sure things are clean when testing  and after

File: tests/test_main.py [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
-    print("hi")
+import pandas as pd
+import pytest
+import tarfile
+
+from video2dataset.main import video2dataset
+
+
+def test_e2e():
+    url_list = "tests/test_files/test_webvid.csv"
```
Comment (rom1504) [inline tests/test_main.py:9]: better get the file relative path (see examples in img2datase tests)

**Comment by rom1504** [comment]
in img2dataset tests I eventually switched to storing files locally and booting a server during tests as external server died

but probably good enough for now

**Pull request merged**